### ISSUE TITLE
It's a Kickstarter and Udemy scam

### ISSUE THREAD

**Issue by DiegoPortoJaccottet** [alexis-jacq/numpy_ARS#3]
They got $136,044 from Kickstarter and God knows how much from Udemy (5,526 students subscribed for $5? Original price was $155. Which equals $27,630-$856,530). By saying things like "Build the Most Powerful AI" and that ARS is able to do the exact same thing as Deep Mind. Just check out their Udemy course "Artificial Intelligence 2018: Build the Most Powerful AI" and watch the preview of their course.

**Comment by alexis-jacq** [comment]
Yes, that's a kind of shame. Hadelin (from Udemy) is a friend and I helped him for some courses content. I know they are overselling this algorithm and exaggerating a bit, but that's their way to sell their courses.  They worked a lot on communication/theater to attract that quantity of students. Just don't buy it, you will learn much more with free moocs from universities and blogs.
